spec/domain/operations/hbx_enrollments/drop_enrollment_members_spec.rb

```diff
@@ -45,35 +46,59 @@
                                          household: family.active_household,
                                          hbx_enrollment_members: [hbx_enrollment_member1, hbx_enrollment_member2, hbx_enrollment_member3],
                                          aasm_state: "coverage_selected",
+                                         kind: "individual",
                                          effective_on: TimeKeeper.date_of_record,
                                          rating_area_id: initial_application.recorded_rating_area_id,
                                          sponsored_benefit_id: initial_application.benefit_packages.first.health_sponsored_benefit.id,
                                          sponsored_benefit_package_id: initial_application.benefit_packages.first.id,
-                                         benefit_sponsorship_id: initial_application.benefit_sponsorship.id)
+                                         benefit_sponsorship_id: initial_application.benefit_sponsorship.id,
+                                         consumer_role_id: consumer_role.id)
       hbx_enrollment.benefit_sponsorship = benefit_sponsorship
       hbx_enrollment.save!
       hbx_enrollment
     end
 
+    before :each do
+      EnrollRegistry[:drop_enrollment_members].feature.stub(:is_enabled).and_return(true)
+    end
+
     context 'when members are selected for drop', dbclean: :around_each do
       context 'when previous enrollment has 0 applied aptc' do
         before do
           @dropped_members = subject.call({hbx_enrollment: enrollment,
-                                           options: {"termination_date_#{enrollment.id}" => TimeKeeper.date_of_record.to_s,
+                                           options: {"termination_date_#{enrollment.id}" => (TimeKeeper.date_of_record + 1.day).to_s,
                                                      "terminate_member_#{hbx_enrollment_member3.id}" => hbx_enrollment_member3.id.to_s}}).success
         end
 
         it 'should return dropped member info' do
           expect(@dropped_members.first[:hbx_id]).to eq hbx_enrollment_member3.hbx_id.to_s
-          expect(@dropped_members.first[:terminated_on]).to eq TimeKeeper.date_of_record
+          expect(@dropped_members.first[:terminated_on]).to eq TimeKeeper.date_of_record + 1.day
         end
 
         it 'should terminate previously existing enrollment' do
           expect(enrollment.aasm_state).to eq 'coverage_terminated'
         end
 
         it 'should select coverage for the reinstatement' do
-          expect(family.hbx_enrollments.last.aasm_state).to eq 'coverage_selected'
+          expect(family.hbx_enrollments.where(:id.ne => enrollment.id).last.aasm_state).to eq 'coverage_selected'
+        end
+      end
+
```

Comment:
Spec missing for some cases can you please add,
Cases like
1.  when subscriber dropped from base enrollment 
2. When term date falls outside current year
3. Rating area changed when dropping dependent
4. product not offered in new service.
please refer old comments for scenario